cmd/root.go

```diff
@@ -62,6 +63,9 @@ func NewCmdRoot(streams genericclioptions.IOStreams) *cobra.Command {
 	templates.ActsAsRootCommand(rootCmd, []string{"options"})
 	rootCmd.AddCommand(newCmdOptions(streams))
 
+	//Add cost command to use AWS Cost Manager
+	rootCmd.AddCommand(cost.CostCmd)
```

Comment:
Could you please make `cost.CostCmd` a function instead of a variable? Just like `cost.NewCmdCost(streams)`.

It would be great if you can add this to L58